arches_references/management/commands/controlled_lists.py

```diff
@@ -218,60 +249,51 @@ def migrate_concept_nodes_to_reference_datatype(self, graph_id):
-                        node.cardxnodexwidget_set.annotate(
-                            config_without_i18n=Cast(
-                                models.F("config"),
-                                output_field=models.JSONField(),
-                            )
+                    cross_records = node.cardxnodexwidget_set.annotate(
+                        config_without_i18n=Cast(
+                            models.F("config"),
+                            output_field=models.JSONField(),
                         )
-                        .annotate(
-                            without_default=CombinedExpression(
-                                models.F("config_without_i18n"),
-                                "-",
-                                models.Value(
-                                    "defaultValue", output_field=models.CharField()
-                                ),
-                                output_field=models.JSONField(),
-                            )
-                        )
-                        .annotate(
-                            without_default_and_options=CombinedExpression(
-                                models.F("without_default"),
-                                "-",
-                                models.Value(
-                                    "options", output_field=models.CharField()
-                                ),
-                                output_field=I18n_JSONField(),
-                            )
+                    ).annotate(
+                        config_without_options=CombinedExpression(
+                            models.F("config_without_i18n"),
+                            "-",
+                            models.Value("options", output_field=models.CharField()),
+                            output_field=I18n_JSONField(),
                         )
                     )
                     for cross_record in cross_records:
                         # work around for i18n as_sql method issue detailed here: https://github.com/archesproject/arches/issues/11473
                         cross_record.config = {}
                         cross_record.save()
 
-                        cross_record.config = cross_record.without_default_and_options
+                        # Crosswalk concept version of default values to reference versions
+                        original_default_value = (
+                            cross_record.config_without_options.get(
+                                "defaultValue", None
+                            )
+                        )
+                        if original_default_value:
+                            new_default_value = []
+                            if isinstance(original_default_value, str):
+                                original_default_value = [original_default_value]
+                            for value in original_default_value:
+                                value_rec = Value.objects.get(pk=value)
```

Comment:
Suggest catching Value.DoesNotExist and reraising as CommandError

Reply:
I don't think this scenario is likely. If a value has been removed from the database but is still used as a default for a node, then the graph is invalid and should be fixed. I'd argue that it's more helpful to have the stack trace from the failed get(). 

One thing we could consider is guarding against a scenario in which the value still exists but the list item value has been removed after the controlled list has been migrated. We already check if the collection has been migrated to a controlled list. However, I think it may be fair to assume that someone hasn't removed list items from the list and that an RDM value and corresponding list item value exist.